src/control/settings/Settings.cpp

```diff
@@ -158,6 +158,7 @@ void Settings::loadDefault() {
     this->strokeFilterIgnoreLength = 1;
     this->strokeFilterSuccessiveTime = 500;
     this->strokeFilterEnabled = false;
+    this->scaleLineWidthEnabled = false;
```

Comment:
You changed the default behaviour of xournal++, this will be very unexpected, resizing written text is not possible without changing it back.

Reply:
Ok, I will set this to true as default.